### PR TITLE
[PR-19369] Update Shopify & Webhook coupon sync documentation

### PR DESCRIPTION
**Pull request by apunko**
Shopify & Webhook coupon auto sync now could be enabled simultaneously.

[DEV PR](https://github.com/talkable/talkable/pull/8808)

## Demo

https://deploy-preview-297--talkable-docs.netlify.app/

## Related Stories

[](https://talkable.atlassian.net/browse/PR-19369)
[](https://talkable.atlassian.net/browse/PR-18755)

### CODE REVIEW

File: source/advanced_features/shopify_coupons_auto_sync.rst [hunk 1]
```diff
@@ -8,12 +8,16 @@ Shopify coupon auto-sync
...
+.. note::
+  Shopify coupon auto-sync and :ref:`Coupon Webhook <web_hooks/create_coupon>` can be enabled simultaneously.
+
 Get started
 -----------
 
 Before using coupon auto-sync, you’ll need to install the Talkable Shopify app (Settings → Shopify Integration → Authorize).
 
-To enable coupon auto-sync in a coupon list, go to the coupon list edit/create page (All reports → Coupon lists) and check "Enable Coupon List Shopify Auto Sync". If this checkbox is disabled (you are unable to check/uncheck), it means this site already uses the :ref:`Coupon Webhook <web_hooks/create_coupon>` and will be created automatically — no coupons can be directly uploaded to Shopify. Instead, coupons are delivered to Coupon Webhook URL, and should be handled from there. If you’d like to turn on/off Shopify auto-sync, you’ll need to first disable the Coupon Webhook.
+To enable coupon auto-sync in a coupon list, go to the coupon list edit/create page (All reports → Coupon lists) and check "Enable Coupon List Shopify Auto Sync".
```
Comment (dgtv) [inline source/advanced_features/shopify_coupons_auto_sync.rst:19]:
```suggestion
To enable coupon auto-sync in a coupon list, go to the coupon list edit/create page (All reports → Coupon lists) and check "Enable Coupon List Shopify Auto-sync".
```

File: source/advanced_features/shopify_coupons_auto_sync.rst [hunk 2]
```diff
@@ -8,12 +8,16 @@ Shopify coupon auto-sync
 
 Talkable supports coupon auto-sync for Shopify. This feature allows the automation of code creation and management of Shopify Price Rules, rather than manual code creation and upload to the Talkable Platform.
 
+.. note::
+  Shopify coupon auto-sync and :ref:`Coupon Webhook <web_hooks/create_coupon>` can be enabled simultaneously.
```
Comment (dgtv) [inline source/advanced_features/shopify_coupons_auto_sync.rst:12]: Lets add a note that webhook will be delivered only after a successful call to Shopify.

File: source/advanced_features/shopify_coupons_auto_sync.rst [hunk 3]
```diff
@@ -8,12 +8,16 @@ Shopify coupon auto-sync
...
+
 Get started
 -----------
 
 Before using coupon auto-sync, you’ll need to install the Talkable Shopify app (Settings → Shopify Integration → Authorize).
 
-To enable coupon auto-sync in a coupon list, go to the coupon list edit/create page (All reports → Coupon lists) and check "Enable Coupon List Shopify Auto Sync". If this checkbox is disabled (you are unable to check/uncheck), it means this site already uses the :ref:`Coupon Webhook <web_hooks/create_coupon>` and will be created automatically — no coupons can be directly uploaded to Shopify. Instead, coupons are delivered to Coupon Webhook URL, and should be handled from there. If you’d like to turn on/off Shopify auto-sync, you’ll need to first disable the Coupon Webhook.
+To enable coupon auto-sync in a coupon list, go to the coupon list edit/create page (All reports → Coupon lists) and check "Enable Coupon List Shopify Auto Sync".
+If this checkbox is disabled (you are unable to check/uncheck), it means that you don't have write rights for this site or Shopify Integration is not authorized.
```
Comment (dgtv) [inline source/advanced_features/shopify_coupons_auto_sync.rst:20]:
```suggestion
If this checkbox is disabled (you are unable to check/uncheck it), it means that your user doesn't have rights for this site or Shopify Integration is not authorized.
```

File: source/advanced_features/shopify_coupons_auto_sync.rst (hunk 2 above)
Comment (apunko) [reply source/advanced_features/shopify_coupons_auto_sync.rst:12]: Done, also added a similar explicit note to Create Coupon Webhook page.

**Review by dgtv — APPROVED** [review]
LG, @zhuravel can you please check English? Thanks

**Review by zhuravel — APPROVED** [review]

**Pull request merged**